From 194671574c2620655a53e5e647048fd0f544678c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 8 Dec 2019 23:35:31 +0100
Subject: [PATCH] luajit: disable for powerpc

It doesn't compile for now.
More details in https://downloads.openwrt.org/releases/faillogs-19.07/powerpc_8540/packages/luajit/compile.txt

--snip--

lj_arch.h:422:2: error: #error "No support for PowerPC CPUs without double-precision FPU"
 #error "No support for PowerPC CPUs without double-precision FPU"
  ^~~~~
lj_arch.h:431:2: error: #error "No support for PPC/e500 anymore (use LuaJIT 2.0)"
 #error "No support for PPC/e500 anymore (use LuaJIT 2.0)"
  ^~~~~

--snip--

We would need to use moonjit fork or
something else. Details are discussed in https://github.com/openwrt/packages/pull/10718

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/luajit/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/luajit/Makefile b/lang/luajit/Makefile
index 296a3ebb6..4fbd5f503 100644
--- a/lang/luajit/Makefile
+++ b/lang/luajit/Makefile
@@ -24,7 +24,7 @@ define Package/luajit
  CATEGORY:=Languages
  TITLE:=LuaJIT
  URL:=https://www.luajit.org
- DEPENDS:=@(i386||x86_64||arm||armeb||aarch64||powerpc||mips||mipsel)
+ DEPENDS:=@(i386||x86_64||arm||armeb||aarch64||mips||mipsel)
 endef
 
 define Package/luajit/description
-- 
2.24.0

